core/src/main/java/org/wildfly/extras/creaper/core/online/ManagementProtocol.java

```diff
@@ -5,6 +5,8 @@
     REMOTE("remote"),
     /** Used in WildFly. Default port 9990. */
     HTTP_REMOTING("http-remoting"),
+    /** Used in WildFly. Default port 9993. */
+    HTTPS_REMOTING("https-remoting"),
```

Comment:
Aha, OK.

Anyway, you'll have to adjust the `if (data.localDefault) { ... }` condition in `OnlineOptions` constructor.